### ISSUE TITLE
Print statement breaks with single wavelength value

### ISSUE THREAD

**Issue by rcooper295** [amerand/CANDID#1]
Hello,
First, thank you for this package -- we used it during the commissioning of JWST AMI mode and it has been very useful.
With the latest update (version 1.0.7, #1a3151c0fdc1484f11fe06c6608d9e94df13c3cc) I find that candid.py lines 1295-1296 break the fitting routine when the EFF_WAVE is a single float, specifically:

```
1295     print(' | EFF_BAND/gradient(EFF_WAVE)', hdu.header['INSNAME'], '~',
-> 1296         '%.3f'%np.mean(hdu.data['EFF_BAND']/np.gradient(hdu.data['EFF_WAVE'])))

ValueError: Shape of array too small to calculate a numerical gradient, at least (edge_order + 1) elements are required.
```
The issue here is taking e.g.  `np.gradient([3.8e-6])` since EFF_WAVE can be a single value for one channel/filter. Could this print statement be altered to accommodate that?
Thank you.

**Comment by amerand** [comment]
Sorry for the missing the problem! I corrected the issue in "master 64dcb17". It should work for single spectral channel (I hope). let me know

**Comment by amerand** [comment]
Looks like the issue is solved
